### PR TITLE
fix/support-ostype-linux-sbtn

### PR DESCRIPTION
**Pull request by Androz2091**
Hello,

This is an attempt of fix for #7894.

### CODE REVIEW

File: sbt [hunk 1]
```diff
@@ -180,7 +180,7 @@ acquire_sbtn () {
   local archive_target=
   local url=
   local arch="x86_64"
-  if [[ "$OSTYPE" == "linux-gnu"* ]]; then
+  if [[ "$OSTYPE" == "linux-gnu"* || "$OSTYPE" == "linux"* ]]; then
```
Comment (eed3si9n) [inline sbt]: Thanks for this. Logically we can get rid of the first check then?

**Review by eed3si9n — CHANGES_REQUESTED** [review]

File: sbt (hunk 1 above)
Comment (Androz2091) [reply sbt]: Yes, indeed, thank you. I'm updating that.

**Review by eed3si9n — APPROVED** [review]
Thanks!

**Comment by eed3si9n** [comment]
@Androz2091 Could you sign the Scala CLA plz? - https://www.lightbend.com/contribute/cla/scala

**Comment by Friendseeker** [comment]
Thanks for the contribution, Simon!

sbt on Linux currently has some quality control issues due to maintainers not using Linux and any improvement on Linux side is very welcomed!

**Review by Friendseeker — APPROVED** [review]

**Comment by Friendseeker** [comment]
> @Androz2091 Could you sign the Scala CLA plz? - https://www.lightbend.com/contribute/cla/scala

I wonder if we still needs the Scala CLA job, now the project is no longer affiliated with Akka (formerly Lightbend)

```
/*
 * sbt
 * Copyright 2023, Scala center
 * Copyright 2011 - 2022, Lightbend, Inc.
 * Copyright 2008 - 2010, Mark Harrah
 * Licensed under Apache License 2.0 (see LICENSE)
 */
```

**Comment by eed3si9n** [comment]
The checker is hosted by lightbend.com, but Scala CLA transfers the copyright and patents explicitly to **EPFL** among other things:

> .Grant of Copyright License. Subject to the terms and conditions of this Agreement, You hereby grant to EPFL and to recipients of software distributed by EPFL a perpetual, worldwide, non-exclusive, no-charge, royalty-free, irrevocable copyright license to reproduce, prepare derivative works of, publicly display, publicly perform, sublicense, and distribute Your Contributions and such derivative works.

See https://www.scala-lang.org/news/2023/08/25/sbt-license-transfer.html.

Technically, by the design of GitHub terms of service section D-6 <https://docs.github.com/en/site-policy/github-terms/github-terms-of-service#6-contributions-under-repository-license>, when you send a pull request, the same licensing terms apply i.e. Apache v2 license:

> Whenever you add Content to a repository containing notice of a license, you license that Content under the same terms, and you agree that you have the right to license that Content under those terms. If you have a separate agreement to license that Content under different terms, such as a contributor license agreement, that agreement will supersede.

Scala CLA makes a stronger claim since the copyright is transferred over. This allows the copyright owners to change the licensing terms (sbt has in the past), and the copyright owner would have legal standing if we need to sue someone for licensing violation I think.

**Comment by Androz2091** [comment]
Thank you for your feedback. I'm having some troubles signing the Scala CLA. When clicking "[Sign the agreement using your github account](https://www.lightbend.com/contribute/cla/scala/sign)", it redirects me to GitHub, then I click "authorize" and gets redirected forever to "https://github.com/login/oauth/authorize?client_id=f26f1ad52ffd85411f9c&state=&scope=" with a different state everytime. Until I get ratelimited by GitHub for approx ~1h. I tried different browsers, to clear the oauth2 authorization from the security settings, even on my phone.
 
image

Can you reproduce the issue, do you want me to report it somehow and get feedback from lightbend?

**Comment by eed3si9n** [comment]
> I'm having some troubles signing the Scala CLA.

@SethTisue Could we get help on this plz? 🙏🏼

**Comment by Friendseeker** [comment]
> > I'm having some troubles signing the Scala CLA.
> 
> @SethTisue Could we get help on this plz? 🙏🏼

Eugene do you by chance have a PDF version of CLA form as a former Akka employee?

> * Alternatively, if you have a scanned, signed CLA form you can email it to [cla@akka.io](mailto:cla@akka.io)

**Comment by eed3si9n** [comment]
I'm not sure if I downloaded CLA pdf.

**Comment by Androz2091** [comment]
Hello, I sent them an email last week and they replied today that the URL is no longer https://www.lightbend.com/contribute/cla/scala/sign but https://contribute.akka.io/contribute/cla/scala. I signed the CLA there, can you confirm it is working? Thank you

**Comment by Friendseeker** [comment]
> Hello, I sent them an email last week and they replied today that the URL is no longer https://www.lightbend.com/contribute/cla/scala/sign but https://contribute.akka.io/contribute/cla/scala. I signed the CLA there, can you confirm it is working? Thank you

`https://www.lightbend.com/contribute/cla/scala/check/Androz2091`

`{"user":"Androz2091","signed":true,"version":"1.0","currentVersion":"1.0"}`

Yay! It works now!

Would you like to retry rebase the branch against latest 1.10.x commit and force push to rerun the CIs?

**Comment by Androz2091** [comment]
@adpi2 thank you for merging ^^

@Friendseeker I've just seen your message, do I still need to do something following the merge?

**Comment by Friendseeker** [comment]
> @adpi2 thank you for merging ^^
> 
> 
> 
> @Friendseeker I've just seen your message, do I still need to do something following the merge?
> 
> 

No. There's nothing that needs to be done on your side. I guess Adrien manually triggered CI so that served the same purpose.

Thanks for the contribution!

**Pull request merged**